docker_environment/docker-compose.yml

```diff
@@ -1,25 +1,29 @@
-version: "3.5"
-
-services:
-    conan-training:
-        container_name: conan-training
-        build:
-            dockerfile: Dockerfile
-            context: ./conan-training
-        networks:
-            - conan-training
-        command: tail -f /dev/null
-
-    jfrog-artifactory-training:
-        container_name: jfrog-artifactory-training
-        build:
-            dockerfile: Dockerfile
-            context: ./artifactory-training
-        networks:
-            - conan-training
-        ports:
-            - "8082:8082"
-        restart: always
-
-networks:
-    conan-training:
\ No newline at end of file
+version: "3.5"
```

Comment:
It changed from CRLF to LF. It's hard to review, but only `CONAN_VERSION: ${CONAN_VERSION}` and `ARTIFACTORY_VERSION: ${ARTIFACTORY_VERSION}` have been added. The advantage is we can parametrize both versions to run locally or for future update.